src/NovaConfigServiceProvider.php

```diff
@@ -28,6 +29,8 @@ class NovaConfigServiceProvider extends ServiceProvider
      */
     public function boot()
     {
+        ConfigItemResource::withoutWrapping();
```

Comment:
This is obvious incorrect solution: such code will disable wrapping for **all** JSON resources in the application.
See plain PHP example:

```php
class A
{
    public static $wrap = 'default';

    public static function setWrap($wrap)
    {
        static::$wrap = $wrap;
    }

    public static function getWrap()
    {
        return static::$wrap;
    }
}

class B extends A {}

B::setWrap('b');

var_dump(A::getWrap()); // outputs: 'b'
```